src/Akeneo/Pim/Structure/Bundle/Resources/public/js/attribute-option/components/Edit.tsx

```diff
@@ -1,10 +1,13 @@
 import React from 'react';
+import {useTranslate} from '@akeneo-pim-community/legacy-bridge';
 
 const Edit = () => {
+    const __ = useTranslate();
```

Comment:
```suggestion
    const translate = useTranslate();
```
As you wish but I think we can get rid of this old convention and use a more appropriate name :)